ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/katex.js

```diff
@@ -17,8 +17,30 @@ katex = mhchemModule(katex);
 function katexStyle() {
 	return [
 		{ name: 'katex.css' },
-		// Note: Katex also requires a number of fonts but they don't need to be specified here
-		// since they will be loaded as needed from the CSS.
+		/* Note: Declaring the fonts is not needed for in-app rendering because
+		they will be loaded automatically from katex.css.
+		However they must be specified for exporting notes to HTML and PDF,
+		so that they can be bundled with the other assets. */
```

Comment:
Did you install the commit hook? I don't think so, otherwise the commet woulf have been changed to:

```
// Note: Declaring the fonts is not needed for in-app rendering because
// they will be loaded automatically from katex.css.
// However they must be specified for exporting notes to HTML and PDF,
// so that they can be bundled with the other assets.
```

Reply:
No I didn't know about commit hooks. Should I manually change it to your format?

> // Note: Declaring the fonts is not needed for in-app rendering because
> // they will be loaded automatically from katex.css.
> // However they must be specified for exporting notes to HTML and PDF,
> // so that they can be bundled with the other assets.

Reply:
No, you should install the commit hook as described in CONTRIBUTING.md and add an empty commit.

Reply:
Yes please, and it's strange it didn't get picked up by CI.

Reply:
> Should I manually change it to your format?

@kowalskidev, recall what I told you once how to read and interpret comments. Your answer puts you quite a bad position. Your talent to read between the lines need still to be improved

Reply:
> @kowalskidev, recall what I told you once how to read and interpret comments. Your answer puts you quite a bad position. Your talent to read between the lines need still to be improved

Oppss!! Sorry sir my bad! I was confused so I simply asked what to do next.
I hope it's all clear now regarding commit hooks.

Reply:
Well, I'm not sure, if you have installed the hook, since you force pushed. Anyway, the comment form is ok, but at the wrong position. see my other comment

Reply:
I think that's an issue with windows. I use dual boot OS (windows and ubuntu) sometime I commit from ubuntu then commit hooks and linter-ci works but in windows I'm not sure it's working properly as it doesn't changed the comments while pushing to GitHub as expected. Next I'll only use ubuntu to make sure everything works fine.

Reply:
The pre-commit hook fires when you commit, not when you push. You can test it locally without having to push. Just create a branch and commit a "wrong" multi line comment.
If the comment has been changed, the hook works.

Reply:
Thank you sir. I'll test it right now.

Reply:
You're welcome. Btw, no need to call me sir.